openverse_catalog/dags/providers/provider_api_scripts/brooklyn_museum.py

```diff
@@ -2,186 +2,163 @@
 
 import lxml.html as html
 from airflow.models import Variable
+from common import constants
 from common.licenses import get_license_info
 from common.loader import provider_details as prov
-from common.requester import DelayedRequester
-from common.storage.image import ImageStore
-from requests.exceptions import JSONDecodeError, RequestException
+from providers.provider_api_scripts.provider_data_ingester import ProviderDataIngester
 
 
-logging.basicConfig(
-    format="%(asctime)s - %(name)s - %(levelname)s:  %(message)s", level=logging.INFO
-)
 logger = logging.getLogger(__name__)
 
-LIMIT = 35
-DELAY = 1.0
-RETRIES = 3
-PROVIDER = prov.BROOKLYN_DEFAULT_PROVIDER
-ENDPOINT = "https://www.brooklynmuseum.org/api/v2/object/"
-API_KEY = Variable.get("API_KEY_BROOKLYN_MUSEUM", default_var="nokeyprovided")
 
-delay_request = DelayedRequester(delay=DELAY)
-image_store = ImageStore(provider=PROVIDER)
+class BrooklynMuseumDataIngester(ProviderDataIngester):
+    providers = {"image": prov.BROOKLYN_DEFAULT_PROVIDER}
+    endpoint = "https://www.brooklynmuseum.org/api/v2/object/"
+    batch_limit = 35
 
-HEADERS = {"api_key": API_KEY}
+    def __init__(self, conf: dict = None, date: str = None):
+        super().__init__(conf, date)
```

Comment:
In some of the other classes, I've been accepting `*args` and passing them all through to the super class, which may be more resistant to breaking with future changes.